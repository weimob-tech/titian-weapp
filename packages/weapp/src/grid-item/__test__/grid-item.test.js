const { getComponent } = require('../../common/test/getComponent');

describe('test component', () => {
  it('should render success', () => {
    const comp = getComponent('grid-item');
    expect(comp).toMatchSnapshot();
  });
});
