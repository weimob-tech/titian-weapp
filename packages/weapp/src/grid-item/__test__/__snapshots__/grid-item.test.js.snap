// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component should render success 1`] = `
<main>
  <wx-view
    class="ext-class titian-grid-item"
    style="width:25%"
    bind:tap="onClick"
  >
    <wx-view
      class="content-class titian-grid-item-content"
      style="flex-direction: column"
      bind:tap="onClick"
    >
      <wx-view />
      <wx-view
        class="text-class titian-grid-item-text"
      />
    </wx-view>
  </wx-view>
</main>
`;
